mucommander-core/src/main/java/com/mucommander/ui/dialog/file/FileCollisionDialog.java

```diff
@@ -55,26 +56,37 @@
  */
 public class FileCollisionDialog extends QuestionDialog {
 
-    /** This value is used by some FileJob classes */
-    public final static int ASK_ACTION = -1;
-	
-    public final static int CANCEL_ACTION = 0;
-    public final static int SKIP_ACTION = 1;
-    public final static int OVERWRITE_ACTION = 2;
-    public final static int OVERWRITE_IF_OLDER_ACTION = 3;
-    public final static int RESUME_ACTION = 4;
-    public final static int RENAME_ACTION = 5;
-
-    public final static String CANCEL_TEXT = Translator.get("cancel");
-    public final static String SKIP_TEXT = Translator.get("skip");
-    public final static String OVERWRITE_TEXT = Translator.get("overwrite");
-    public final static String OVERWRITE_IF_OLDER_TEXT = Translator.get("overwrite_if_older");
-    public final static String RESUME_TEXT = Translator.get("resume");
-    public final static String RENAME_TEXT = Translator.get("rename");
+    public enum OverwriteAction implements DialogAction {
```

Comment:
how about 'CollisionAction' or 'FileExistsAction' instead of 'OverwriteAction'?  it's a bit weird to see OverwriteAction.OVERWRITE